lib/components/artist/related_artists/related_artist.js

```diff
@@ -0,0 +1,67 @@
+/* @flow */
+'use strict';
+
+import Relay from 'react-relay';
+import React from 'react-native';
+const { ScrollView, StyleSheet, View, Text } = React;
+
+import colors from '../../../../data/colors';
+import SerifText from '../../text/serif';
+import ImageView from '../../opaque_image_view';
+
+class RelatedArtist extends React.Component {
+  render() {
+    const artist = this.props.artist;
+    return (
+      <View style={{ width: 300 }}>
+        <ImageView style={{ height: 175, marginRight: 20 }}
+                   aspectRatio={1.7}
+                   imageURL={artist.image.url} />
+        <Text style={styles.sansSerifText}>{artist.name.toUpperCase()}</Text>
+        <Text style={styles.serifText}>{this.artworksString(artist)}</Text>
+      </View>
+    )
+  }
+
+  artworksString(artist) {
+    const forSale = artist.counts.for_sale_artworks ? artist.counts.for_sale_artworks + ' for sale' : null
```

Comment:
Seems like you really only need to pass in `artist.counts` instead of the complete `artist`.